incubator/gitea/templates/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "gitea.fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: {{ template "gitea.fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "gitea.fullname" . }}
+    spec:
+      initContainers:
+        - name: copy-app-ini
+          image: alpine:3.7
+          command: ["/bin/sh"]
+          args: ["-c", "cp /config/* /config-rw/ ; chmod 777 /config-rw/*"]
+          volumeMounts:
+            - name: config
+              mountPath: /config
+            - name: config-rw
+              mountPath: /config-rw
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - containerPort: 3000
+            - containerPort: {{ .Values.service.sshPort | int }}
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 3000
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 3000
+          env:
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
+          volumeMounts:
+            - name: data
+              mountPath: /data
+            - name: config-rw
+              mountPath: /etc/gitea/conf
```

Comment:
I suspect you mean to mount this to `/etc/templates/` so that it gets consumed by https://github.com/go-gitea/gitea/blob/6e03390aa8fa096206457962db1955d642860b57/docker/etc/s6/gitea/setup#L39 on initial start up